docs/wire_compiler.md

```diff
@@ -318,8 +319,13 @@ wire {
     // target.
     javaInterop = true
 
-    // True for emitted services to provide blocking APIs only.
-    blockingServices = true
+    // SUSPENDING to generate coroutines APIs that require a Kotlin coroutines context.
+    // BLOCKING to generate blocking APIs callable by Java and Kotlin.
+    rpcCallStyle = com.squareup.wire.kotlin.RpcCallStyle.BLOCKING
```

Comment:
With no auto-complete in Groovy I expect these to be painful to use.. Should we introduce overloads that accept Strings?
```groovy
wire {
  kotlin {
    rpcCallStyle = 'blocking'
    rpcRole = 'server'
  }
}
```

Reply:
What’s idiomatic?

Reply:
I’m gonna land as-is and add more later.

Reply:
Yeah, not sure what the idiomatic way is.

Reply:
definitely a string. you can do `values().singleOrNull { it.equals(whatever, ignoreCase=true) } ?: throw IllegalArgumentException("Unknown whatever $whatever. Valid values: ${values().contentToString()}")`